packages/nextjs-mf/utils/build-utils.ts

```diff
@@ -1,5 +1,14 @@
 import path from 'path';
 
+/**
+ * Computes the remote filename based on the server status and the provided filename.
+ * If the server is active and a filename is provided, it returns the basename of the filename.
+ * Otherwise, it returns the provided filename.
+ *
+ * @param {boolean} isServer - Indicates if the server is active.
```

Comment:
The function 'computeRemoteFilename' does not handle the case when the filename is null or undefined. This could lead to unexpected behavior or errors. Add a check to handle such cases.